kwave/executor.py

```diff
@@ -41,6 +41,7 @@ def run_simulation(self, input_filename: str, output_filename: str, options: str
 
         return sensor_data
 
+    # TODO (Walter): make static
```

Comment:
Do you want to leave this todo item for future or address it before merging this PR (fine by me either way)?